test/test/AppDelegate.swift

```diff
@@ -13,8 +13,8 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
     var window: UIWindow?
 
-
-    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
+    @nonobjc
```

Comment:
This is actually in app, not specifically for travis. We were just getting a compiler warning that was making the travis build fail. I think when we tell travis that it is objc it just defines the vm to use and stuff, the annotation is specifically for the swift compiler. If that makes sense.